puppet/modules/chassis/manifests/wp.pp

```diff
@@ -11,6 +11,8 @@
 	$admin_email    = 'admin@example.com',
 	$admin_password = 'password',
 	$network = false,
+	$plugins = [],
+	$theme = '',
```

Comment:
Unless you're talking about installing wp.org themes ( pretty rare I guess with such setup ), you wouldn't really need to do this since you'd be cloning / sub modeling them within content folder. Do you think we should cater for this edgy scenario ?